src/ape/_pydantic_compat.py

```diff
@@ -27,7 +29,6 @@
         validator,
     )
     from pydantic.dataclasses import dataclass  # type: ignore
```

Comment:
Just noticed that this is not in the v1 import

Reply:
done [c95e7c5](https://github.com/ApeWorX/ape/pull/1651/commits/c95e7c50fdd0e76b585f32230305f325a5d6dacf)